package.json

```diff
@@ -66,9 +67,6 @@
         "process-spectrum": "wireit",
         "publish:react": "yarn changeset publish --no-git-tag --tag snapshot --no-push",
         "push-to-remote": "git add . && git commit -m \"chore: release new versions #publish\" && git push",
-        "spectrum-css": "wireit",
-        "spectrum-tokens": "wireit",
-        "spectrum-vars": "wireit",
```

Comment:
These commands don't exist in wireit and haven't for a long while - seems safe to remove the alias'es right?